zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/ZeebeFormDefinitionValidator.java

```diff
@@ -56,6 +60,14 @@ public void validate(
                 ZeebeConstants.ATTRIBUTE_FORM_KEY,
                 ""));
       }
+      if (!isBlank(formKey) && bindingType != null) {
+        validationResultCollector.addError(
+            0,
+            String.format(
+                ERROR_MESSAGE_INVALID_ATTRIBUTE_COMBINATION,
+                ZeebeConstants.ATTRIBUTE_FORM_KEY,
+                ZeebeConstants.ATTRIBUTE_BINDING_TYPE));
+      }
```

Comment:
I'm not sure whether such a validation is actually needed (the `bindingType` could also just be ignored if it was used with a `formKey` or `externalReference`). But I think it doesn't hurt to be strict here and return an error.

Reply:
🔧 With my previous comment in mind, it may be better to allow `bindingType` and `formKey` at the same time, to allow a default `bindingType` value.

Reply:
Yes, if we decide to always set a default value for `bindingType`, let's just remove this extra validation.

One use case I had in mind when adding it was that users might have specified the wrong attribute by mistake (e.g. they actually wanted to use `formId` with `bindingType=deployment`, but used `formKey` instead). But that would probably be quite an edge case.